src/main/java/frc/robot/RobotContainer.java

```diff
@@ -133,10 +134,10 @@ private void initSubsystems() {
           case "ClimbSubsystem":
             {
               System.out.println("Climb enabled");
-
-              climbSubsystem = new ClimbSubsystem();
-              climbUPCommand = new ClimbUPCommand(climbSubsystem);
-              climbDOWNCommand = new ClimbDOWNCommand(climbSubsystem);
+              if (!Constants.oneController) {
+                climbSubsystem = new ClimbSubsystem(driverJoystick2);
+              }
```

Comment:
need an else case otherwise the subsystem won't get created if oneController is true